foam_controlwrapper/foam_mesh.py

```diff
@@ -181,8 +184,7 @@ def __init__(self, name, BC=None, mesh=None, path=None):
             foamface.writeMesh(name)
 
             # write possible cell data to time directory
-            for cell in mesh.iter_cells():
-                self.update_cell(cell)
+            self.update_cells(list(mesh.iter_cells()))
```

Comment:
list removed and corresponding change made on update_cells method.
update_cells() call is inside 
 if mesh: